src/modules/producer.js

```diff
@@ -170,12 +171,16 @@ class Producer {
       if (!msg) msg = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.dialonce.io/coding_rules#rule_key=javascript%3AOneStatementPerLine)